src/Fixer/ClassNotation/ClassDefinitionFixer.php

```diff
@@ -457,4 +472,45 @@ private function makeClassyInheritancePartMultiLine(Tokens $tokens, int $startIn
             $i = $previousInterfaceImplementingIndex + 1;
         }
     }
+
+    /**
+     * @param array{
+     *     start: int,
+     *     classy: int,
+     *     open: int,
+     *     extends: false|array{start: int, numberOfExtends: int, multiLine: bool},
```

Comment:
It is because the rule fixes all kinds if `classy` definitions, so `class`, `trait` and `interface`.
Interfaces can have multiple `extends`, for example in the test of this fixer we have

```php
<?php interface TestWithMultiExtendsMultiLine extends A,\nAb,C,D\n{}
```

as test input

Reply:
Ah ok, I missed it. Thanks!